dags/pensieve.py

```diff
@@ -0,0 +1,60 @@
+from airflow import DAG
+from airflow.operators.sensors import ExternalTaskSensor
+from datetime import timedelta, datetime
+from operators.gcp_container_operator import GKEPodOperator
+from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
+
+default_args = {
+    'owner': ['ascholtz@mozilla.com', 'sunahsuh@mozilla.com', 'tdsmith@mozilla.com'],
+    'depends_on_past': False,
+    'start_date': datetime(2020, 03, 9),
+    'email_on_failure': True,
+    'email_on_retry': True,
+    'retries': 2,
+    'retry_delay': timedelta(minutes=30),
+}
+
+with DAG('pensieve',
+         default_args=default_args,
+         schedule_interval='@daily') as dag:
+
+    # Built from repo https://github.com/mozilla/pensieve
+    pensieve_image='gcr.io/moz-fx-data-experiments/pensieve:latest'
+
+    pensieve = GKEPodOperator(
+        task_id="pensieve",
+        name='pensieve',
+        image=pensieve_image,
+        email=['ascholtz@mozilla.com', 'sunahsuh@mozilla.com', 'tdsmith@mozilla.com'],
+        dag=dag)
+
+    wait_for_clients_daily_export = ExternalTaskSensor(
+        task_id="wait_for_clients_daily_export",
+        external_dag_id="main_summary",
+        external_task_id="clients_daily_export",
```

Comment:
I think all of these jobs are actually scheduled to start at 01:00 UTC (using a schedule_interval of `0 1 * * *`) so these task sensors either need to include an execution delta like [this](https://github.com/mozilla/telemetry-airflow/blob/02ab6b7d489e7de545913c46938a5c85fb7e07db/dags/addons_daily.py#L31) or (and I'd recommend this just to keep things simple) we change the dag's schedule_interval from `@daily` to `0 1 * * *`